app/models/order_address.rb

```diff
@@ -0,0 +1,20 @@
+class OrderAddress
+  include ActiveModel::Model
+  attr_accessor :postal_code, :prefecture_id, :city, :house_number, :building_name, :phone_number, :user_id, :item_id, :token
+
+  with_options presence: true do
+    validates :postal_code, format: {with: /\A[0-9]{3}-[0-9]{4}\z/, message: "is invalid. Include hyphen(-)"}
+    validates :prefecture_id, numericality: {other_than: 0, message: "can't be blank"}
```

Comment:
```suggestion
    validates :prefecture_id, numericality: {other_than: 1, message: "can't be blank"}
```
prefecture.rb内を確認すると、`{ id: 1, name: '--' }`と記述されていますので、
other_than: 0からother_than: 1へ修正しましょう。